conf/st2.dev.conf

```diff
@@ -5,7 +5,6 @@
 host = 0.0.0.0
 port = 9101
 logging = st2api/conf/logging.conf
-serve_webui_files = True
 # allow_origin is required for handling CORS in st2 web UI.
 # allow_origin = http://myhost1.example.com:3000,http://myhost2.example.com:3000
```

Comment:
I think we just set allow_origin = * during st2_deploy install and call it a day.  st2_deploy shouldn't be used for production deployments (they should use apache or nginx anyway, as you mentioned) so it should be fine, I think.